app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/tax_due.scala.html

```diff
@@ -101,25 +101,45 @@
     <ul class="calc-rows with-bottom-margin">
       <li>
         @calcRow(calculatedTax.initialGainOrLoss.amount, messages(s"$key.initialGainOrLoss"), showGainType = true)
-        @details(messages("calculator.showWorkings"), initialGainOrLossWorkings(disposalDetails, acquisitionDetails, calculatedTax))
+        @details(
+          messages("calculator.showWorkings"),
+          initialGainOrLossWorkings(disposalDetails, acquisitionDetails, calculatedTax),
+          Some(s"${messages("generic.for")} ${messages(s"$key.initialGainOrLoss")}")
```

Comment:
doesn't this assume that the grammatical structure of the sentence will be the same in different languages. Not sure how Welsh works, but to future proof it it'd be safer to have the whole message under a single key

Reply:
I have combined "for" into single key for the assistive text